hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -2418,15 +2418,15 @@ public void testSqlSourceSource() throws Exception {
   @Disabled
   @Test
   public void testJdbcSourceIncrementalFetchInContinuousMode() {
-    try (Connection connection = DriverManager.getConnection("jdbc:h2:mem:test_mem", "test", "jdbc")) {
+    try (Connection connection = DriverManager.getConnection("jdbc:h2:mem:test_mem", "sa", "")) {
```

Comment:
Looks like this is from another fix.  Could the JDBC user and password be put into static variables to avoid any misconfigs in tests in the future?

Reply:
Yes, it's from another fix. It's really better to do it in separate PR later. For now - reverted JDBC user name and password changes.